piv/src/main/java/com/yubico/yubikit/piv/PivSession.java

```diff
@@ -128,6 +128,11 @@ public boolean isSupportedBy(Version version) {
      */
     public static final Feature<PivSession> FEATURE_MOVE_KEY = new Feature.Versioned<>("Move or delete keys", 5, 7, 0);
 
+    /**
+     * Support for the large key sizes in Yubikey 5.7 and onwards.
```

Comment:
To have consistent comments, I would change this comment to be specific and remove the fw version info. This text will show in IDE's on hower. Example how it looks for other `FEATURE`:

![image](https://github.com/Yubico/yubikit-android/assets/1954891/3addd619-788d-4c88-a571-d44ce7c93157)